### ISSUE TITLE
Ignore built-in globals that produce false-positives

### ISSUE THREAD

**Issue by EvgenyOrekhov** [EvgenyOrekhov/eslint-config-hardcore#113]
These produce false positives in browser environment:

```json
{
  "globals": {
    "blur": "off",
    "defaultstatus": "off",
    "defaultStatus": "off",
    "event": "off",
    "find": "off",
    "length": "off",
    "name": "off",
    "status": "off",
    "top": "off"
  }
}
```

**Comment by EvgenyOrekhov** [comment]
Should fine-tune the `no-shadow` rule:

```text
        "no-shadow": [
          "error",
          { "builtinGlobals": true, "allow": ["name", ...] }
        ],
```
